web/next.config.js

```diff
@@ -1,6 +1,33 @@
+const client = require('./lib/sanity.server');
+
 module.exports = {
   reactStrictMode: true,
   images: {
     domains: ['cdn.sanity.io'],
   },
+  async redirects() {
+    // ".microcopy[key == "mainCta"]" filtering does not work, so we have to filter server side
```

Comment:
Nit: I think this might work with a newer version of the API. Probably not a good idea to upgrade such a thing at this point, but might be worth a TODO or issue?